L/libsingular_julia/build_tarballs.jl

```diff
@@ -50,7 +50,7 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    BuildDependency("libjulia_jll"),
+    BuildDependency(PackageSpec(;name="libjulia_jll", version=v"1.10.6")),
     BuildDependency("GMP_jll"),
     BuildDependency("MPFR_jll"),
     Dependency("libcxxwrap_julia_jll"; compat = "~0.11.0"),
```

Comment:
```suggestion
    Dependency("libcxxwrap_julia_jll"; compat = "~0.11.1"),
```

Reply:
So it keeps building against 0.11.0. Alas, if I change the compat here, then this JLL also need a version increment...

Reply:
I will be on the road now and can take of that later, if anyone else wants to modify this PR in the meantime, be my guest ;-)